### PR TITLE
aaspe-common init

### PR DESCRIPTION
**Pull request by martafullen**
New repository for the common components shared by eclipse-aaspe and aasx-server

### CODE REVIEW

File: otterdog/eclipse-aaspe.jsonnet [hunk 1]
```diff
@@ -99,5 +99,18 @@ orgs.newOrg('eclipse-aaspe') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('aaspe-common') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "aaspe common components",
+      has_wiki: false,
+      secret_scanning: "disabled",
```
Comment (netomi) [inline otterdog/eclipse-aaspe.jsonnet]: secret_scanning and push_protection should ideally be enabled for all repos.

See also https://otterdog.eclipse.org/projects/dt.aaspe for information about your current status and configuration.

Comment (martafullen) [reply otterdog/eclipse-aaspe.jsonnet]: thanks, will amend the config

**Comment by netomi** [comment]
LGTM, can a project lead approve? ty

**Review by aorzelskiGH — APPROVED** [review]
Looks good to me

**Review by netomi — APPROVED** [review]

**Pull request merged**